cpp/inc/bond/protocol/simple_json_reader.h

```diff
@@ -108,6 +108,13 @@ class SimpleJsonReader
     {
         return _value == rhs._value;
     }
+
+    /// @brief Access to underlaying buffer
+    typename boost::call_traits<Buffer>::reference
```

Comment:
Didn't realize that `SimpleJsonReader` doesn't have `const` version of `GetBuffer`. Do you mean to add them for _all_ writers as well (none of them has now)?